### PR TITLE
[DOCS] #4941 – FAQ Borg config dir

### LINKED ISSUE #4941: How important is $HOME/.config/borg?
I've been reading the docs and issues over and over and it isn't clear to me how important the `$HOME/.config/borg` folder is. I'm getting the impression that analogous to duplicacy's per-repo `config` file, that the `$HOME/.config/borg` folder needs to be backed up independently as it contains stateful information. Which leads me to the following questions and I apologize if this has already been answered elsewhere:

- Should I back this folder up independently? i.e, if it gets corrupted what do I lose in terms of the ability to do with my repos?

- Is it a security risk if someone gains access to this folder? Meaning can I back it up in plaintext or do I need to encrypt it (file permissions aside)?

- Furthermore what happens if you delete `$HOME/.config/borg/cache`. Considering it's called the "cache" folder one would think it could be deleted safely, but it appears that if you delete it you need to use environment variables to silence warnings about unknown/untrusted repositories. Is this permanent or can the cache be regenerated?

### PR DESCRIPTION
**Pull request by fantasya-pbem**
fixes #4941

### CODE REVIEW

File: docs/usage_general.rst.inc [hunk 1]
```diff
@@ -230,8 +230,6 @@ General:
             in systemd.nspawn containers on some architectures (e.g. ARM).
             Using this does not affect data safety, but might result in a more bursty
             write to disk behaviour (not continuously streaming to disk).
-    TMPDIR
```
Comment (fantasya-pbem) [inline docs/usage_general.rst.inc]: This is moved down to Directories section.

File: docs/faq.rst [hunk 2]
```diff
@@ -353,6 +353,28 @@ to change them.
...
+  If a repository's status is different from this information at the beginning of BorgBackup
+  operation, Borg outputs warning messages and asks for confirmation, so make sure you do not lose
+  or manipulate these files. However, apart from that warnings, a loss of these files can be
+  recovered.
+
+``keys`` subdirectory
+  In this directory all your repository keyfiles are stored. You MUST make sure to have an
+  independent backup of these keyfiles, otherwise you cannot access your backups anymore if you lose
+  them. You also MUST keep these files secret; everyone who gains access to your repository and has
+  the corresponding keyfile (and the protection password) can extract it.
```
Comment (ThomasWaldmann) [inline docs/faq.rst]: s/protection password/key passphrase/ - just to get the terminology in sync with other places.

File: docs/faq.rst [hunk 3]
```diff
@@ -353,6 +353,28 @@ to change them.
...
+How important is the $HOME/.config/borg directory?
+--------------------------------------------------
+
+The Borg config directory has content that you should take care of:
+
+``security`` subdirectory
+  Each directory here represents one Borg repository by its ID and contains the last known status.
+  If a repository's status is different from this information at the beginning of BorgBackup
+  operation, Borg outputs warning messages and asks for confirmation, so make sure you do not lose
+  or manipulate these files. However, apart from those warnings, a loss of these files can be
```
Comment (fantasya-pbem) [inline docs/faq.rst]: s/that/those/ warnings

File: docs/faq.rst [hunk 4]
```diff
@@ -353,6 +353,28 @@ to change them.
...
+  recovered.
+
+``keys`` subdirectory
+  In this directory all your repository keyfiles are stored. You MUST make sure to have an
+  independent backup of these keyfiles, otherwise you cannot access your backups anymore if you lose
+  them. You also MUST keep these files secret; everyone who gains access to your repository and has
+  the corresponding keyfile (and the key passphrase) can extract it.
```
Comment (fantasya-pbem) [inline docs/faq.rst]: s/protection password/key passphrase/

**Comment by ThomasWaldmann** [comment]
Thanks!

**Pull request merged**